balancer/internal/wrr/random.go

```diff
@@ -0,0 +1,58 @@
+/*
+ *
+ * Copyright 2019 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package wrr
+
+import "google.golang.org/grpc/internal/grpcrand"
+
+// randWeighted is a wrapped weighted item that is used to implement weighted random algorithm.
+type randWeighted struct {
```

Comment:
can we remove the "rand" prefix, and name it e.g. `weightedItem`? The struct itself is unrelated to randomness, it's just you currently only used it for the `randomWRR`.

Reply:
Done